CMakeLists.txt

```diff
@@ -28,7 +28,7 @@ include(os_release_info.cmake)
 get_os_release_info(os_name os_version)
 
 if(NOT DEFINED CMAKE_INSTALL_PREFIX OR CMAKE_INSTALL_PREFIX STREQUAL "")
-    set(CMAKE_INSTALL_PREFIX "/usr/local" CACHE PATH "..." FORCE)
+    set(CMAKE_INSTALL_PREFIX "/usr/lib/x86_64-linux-gnu/" CACHE PATH "..." FORCE)
```

Comment:
I do not think that any manual adjustments of CMAKE_INSTALL_PREFIX should be here at all. Cmake does have defaults for this stuff which depends on the underlying OS (one default for Debian/Ubuntu, another for Fedora, etc.). If this default is not being picked up, then most probably something is just wrong in the cmake scripts. Most likely the following include should be done before doing anything with prefix:
```
include(GNUInstallDirs)
```

Reply:
Agreed that defaults should be avoided, I also like the idea of using `include(GNUInstallDirs)`.

Specifically on GNUInstallDirs, It's currently used/called at +56, after we set the users preferred config.  This may be the source of some of the issues. 


As for what the last case scenario default should be, since the docs target Ubuntu for their defaults configurations, following the Ubuntu standard seems reasonable.